src/audio/component.c

```diff
@@ -65,9 +65,14 @@ struct comp_dev *comp_new(struct sof_ipc_comp *comp)
 	/* find the driver for our new component */
 	drv = get_drv(comp->type);
 	if (!drv) {
-		tr_err(&comp_tr, "comp_new(): driver not found, comp->type = %u",
-		       comp->type);
-		return NULL;
+		tr_info(&comp_tr, "comp_new(), no type %d, fallback to use subtype",
+			comp->type);
```

Comment:
I think this should be tr_dbg instead of tr_info, when we will have full support for traces this will make a difference.

Reply:
I wanted this message to be logged most of the time, that's why I uses tr_info, not tr_dbg.